### ISSUE TITLE
Create slide image

### ISSUE THREAD

**Issue by adamshakhabov** [ShapeCrawler/ShapeCrawler#344]
### Discussed in https://github.com/ShapeCrawler/ShapeCrawler/discussions/229

<div type='discussions-op-text'>

<sup>Originally posted by **egilr** February 11, 2022</sup>
Is it possible to get an image (PNG) from the complete slide. </div>

**Comment by sanjevirau** [comment]
Hi @ashahabov, I am just curious to know how are you planning to implement this feature. I have been following ShapeCrawler and using the OpenXML library in my prior projects. 

As far as I know, there are no easy methods to do this, unless we recreate the entire slide image using the shape colors, coordinates, and so on using a drawing library like [SkiaSharp](https://github.com/mono/SkiaSharp). As for the .NET Core 6.0, Microsoft has also dropped supporting Drawing.Common for non-Windows platforms.

Just curious to know, how can we make SharpeCrawler achieve PNG or JPEG slide export in the future.

**Comment by ashahabov** [comment]
Hello @sanjevirau,

Thank you for your feedback! Sorry for the delayed response.

You're right, it's not simple.

In short, there are two ideas:

- **Using SkiaSharp**: Draw each element individually. Yes, we need to draw every detail: coordinates, borders, fonts, colors, etc.
- or **Convert to HTML first**, then use a library to generate an image from HTML. I thought creating HTML might be easier.

I've tried both methods a bit. I'm not sure if generating HTML for the slide is easier than directly converting it to an image with SkiaSharp.

**Comment by jcoliz** [comment]
It's work considering converting to SVG. That may be a closer representation to the PowerPoint primitives. Then user could convert to a raster format using many tools, and/or embed in HTML.

**Comment by ashahabov** [comment]
Yeah, initial conversion to SVG can be considered, too. I haven't had time to try [**SVG**](https://github.com/svg-net/SVG) and SkiaSharp yet. We need a volunteer🙂
